index.html

```diff
@@ -41,5 +41,141 @@ <h4>LET'S CONNECT</h4>
         </ul>
      </div> 
   </section> 
+  <section class="project-1">
```

Comment:
- [ ] I recommend you to use a `div` container to wrap your project card instead of `section` for more semantic reason, the `<section>` tag is used to wrap the whole projects section not just an element.

```
<section>
  <div class="project-card">...</div>
  <div class="project-card">...</div>
  <div class="project-card">...</div>
  <div class="project-card">...</div>
 ....
</section>
```

- [ ] the Project Card should have a white background as you can see in the screenshots below, Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/153614747-d0cdfb32-96ac-4b88-9916-d92b5330a8d8.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/153614752-658ab354-1b42-4f7c-aeb8-b9d2ff37c21b.PNG)

Reply:
@Hamzaoutdoors Hi Hamza
Thank you for your valuable review.
I had a doubt, shall I add .vscode  or ``.vscode` to .gitignore file ?

Reply:
Hello @Shubh-Dev, add it as you added your node_modules folder to your .gitignore because the .vscode is also a folder :) in this case, it would be: .vscode/

Happy coding!